client-go/cmd/releases.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/deis/deis/client-go/controller/models/releases"
+)
+
+// ReleasesList lists an app's releases.
+func ReleasesList(appID string) error {
+	c, appID, err := load(appID)
+
+	if err != nil {
+		return err
+	}
+
+	releases, err := releases.List(c, appID)
+
+	fmt.Printf("=== %s Releases\n", appID)
+
+	w := new(tabwriter.Writer)
+
+	w.Init(os.Stdout, 0, 8, 1, '\t', 0)
+	for _, r := range releases {
+		fmt.Fprintf(w, "v%d\t%s\t%s\n", r.Version, r.Created, r.Summary)
+	}
+	w.Flush()
+	return nil
+}
+
+// ReleasesInfo prints info about a specific release.
+func ReleasesInfo(appID string, version int) error {
+	c, appID, err := load(appID)
+
+	if err != nil {
+		return err
+	}
+
+	r, err := releases.Get(c, appID, version)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== %s Release v%d\n", appID, version)
+	if r.Build != "" {
+		fmt.Println("build:   ", r.Build)
```

Comment:
Nothing else like this is capitalized, so it would be a client wide change. :smiley: